### DISCUSSION TITLE
Height is not being reported

### DISCUSSION THREAD

**Discussion by croogie** [gcormier/megadesk#160] [Q&A]
I have a 3-pin mega desk board. I also bought a companion board. Everything is connected and working almost well:
![image](https://github.com/gcormier/megadesk/assets/33229/ceb88b1c-abb8-4e9e-8862-9f0f131d4a07)

I compiled and installed default `yaml` which is in the repo.

The thing is that I cannot get height updated every time desk changes it's position. It reports `0` all the time: 
![image](https://github.com/gcormier/megadesk/assets/33229/5693d765-1d99-4e3b-bf3c-90f8516b7932)

Do you know if I missed anything in configuration or shell I do something additional to have it working?

**Comment by gcormier** [comment]
Can you try swapping the RX/TX pins in the yaml and reflashing?

**Reply by croogie** [reply]
I switched them but desk was not responding. Getting back to original values reverted to the state I described above.

**Reply by gcormier** [reply]
How old is your 3 pin megadesks? Do you have the serial firmware on it?

**Reply by croogie** [reply]
hmmm - hard to say. Don't remember to be honest. I bought it while ago then after about a year I bought companion. Can you remind me how I can check it?

**Reply by gcormier** [reply]
December 2023 is when I started flashing them with serial firmware - looks like yours was from December 2022. So you will need to re-flash it with the firmware with serial enabled.

https://github.com/gcormier/megadesk/releases/download/v2022.09/megadesk-v2022.09-t841-serial.hex
https://github.com/gcormier/megadesk/blob/master/DIY.md#programming

There are lots of discussions on here about flashing as well.

**Reply by croogie** [reply]
ok - I will try to flash it again 🤞

**Comment by croogie** [comment]
I'm closing the discussion as I burned up the board by mistake 😱
New one arrived - everything working fine!
